ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -807,6 +821,10 @@ import '../nuxeo-button-styles.js';
       }
     }
 
+    _isSelectionEnabled() {
```

Comment:
Where is this being used?

Reply:
I think it is a helper that I must have created and then I should have removed 😬 (so, not being used at all).